h/activity/query.py

```diff
@@ -130,7 +131,8 @@ def execute(request, query, page_size):
             for annotation in bucket.annotations:
                 bucket.presented_annotations.append({
                     'annotation': presenters.AnnotationHTMLPresenter(annotation),
-                    'group': groups.get(annotation.groupid)
+                    'group': groups.get(annotation.groupid),
+                    'incontext_link':links.incontext_link(request, annotation)
```

Comment:
PEP 8, there should be a space here: `'incontext_link': links.incontext_link(request, annotation)`